docs/source/user-guide/expressions.md

```diff
@@ -44,6 +44,15 @@ The following Spark expressions are currently available. Any known compatibility
 | IntegralDivide (`div`) | All operands are cast to DecimalType (in case the input type is not already decima type) with precision 19 and scale 0. Please set `spark.comet.cast.allowIncompatible` to `true` to enable DataFusion’s cast operation for LongType inputs. |
 | Remainder (`%`)        |                                                                                                                                                                                                                                               |
 
+## Binary Try Arithmetic
+
+| Expression | Notes                                                                      |
+|------------|----------------------------------------------------------------------------|
+| `try_add`  | Adds operands (IntegerTypes only) or results NULL incase of overflow       |
+| `try_sub`  | Subtracts operands (IntegerTypes only) or results NULL incase of overflow  |
+| `try_mul`  | Multiplies operands (IntegerTypes only) or results NULL incase of overflow |
+| `try_div`  | Subtracts operands (IntegerTypes only) or results NULL incase of overflow  |
```

Comment:
Question if Comet encounters try_add with non-integer types, do we safely fallback to Spark?

Reply:
The only numeric datatypes spark supports with NULL on Overflow are IntegerTypes (int / long -> int32 /int64) . Along with that Spark supports try arithmetic for datetime literals as well but that isnt supported in comet yet and fallsback to spark

Reply:
If it falls back to Spark, then it's fine, but I was unable to find the code that checks the data type. I wonder what will happen with float/double/decimal...etc